api/XMLHttpRequest.json

```diff
@@ -1139,7 +1139,7 @@
                 "version_added": "7"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "7"
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/XMLHttpRequest/responseType tests false in iOS 7 on a real BrowserStack device. Let me first fix the test to use an instance...